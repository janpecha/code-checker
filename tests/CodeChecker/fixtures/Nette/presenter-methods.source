<?php

	namespace MyApp\Presenters;


	class Test extends \Nette\Application\UI\Presenter
	{
		public function actionDefault()
		{
			if ($this->isPost()) {
				$this->terminate();
			}
		}
	}
