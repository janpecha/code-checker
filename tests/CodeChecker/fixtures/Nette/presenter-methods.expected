<?php

	namespace MyApp\Presenters;


	class Test extends \Nette\Application\UI\Presenter
	{
		public function actionDefault()
		{
			if ($this->isMethod('POST')) {
				$this->terminate();
			}
		}
	}
